doc/manuals/orion-api.md

```diff
@@ -1064,21 +1064,23 @@ are aware of the format without needing to process the notification payload.
 
 ## Custom Notifications
 
-NGSIv2 clients can customize HTTP notification messages using a simple template mechanism. The
+NGSIv2 clients can customize notification messages using a simple template mechanism. The
 `notification.httpCustom` property of a subscription allows to specify the following fields
-to be templatized:
+to be templatized when using HTTP notifications:
 
 * `url`
 * `headers` (both header name and value can be templatized). Note that `Fiware-Correlator` and
   `Ngsiv2-AttrsFormat` headers cannot be overwritten in custom notifications. Any attempt of 
   doing so (e.g. `"httpCustom": { ... "headers": {"Fiware-Correlator": "foo"} ...}` will be ignored.
 * `qs` (both parameter name and value can be templatized)
 * `payload`
-
-The fifth field `method`, lets the NGSIv2 clients select the HTTP method to be used for delivering
+* `method`, lets the NGSIv2 clients select the HTTP method to be used for delivering
 the notification, but note that only valid HTTP verbs can be used: GET, PUT, POST, DELETE, PATCH,
 HEAD, OPTIONS, TRACE, and CONNECT.
 
+Regarding the MQTT notifications, the `mqttCustom` is used instead of `httpCustom`. This
+topic is described with more detail [in this specific document](user/mqtt_notifications.md).
+
```

Comment:
The text in that part of the document pointing can be embedded here: 

```markdown
Custom notifications (described in the [NGSIv2 specification](http://telefonicaid.github.io/fiware-orion/api/v2/stable))
in MQTT subscriptions work the same as in HTTP subscriptions, taking into account the following:

* `mqttCustom` is used instead of `httpCustom`
* The same fields used in `mqtt` can be used in `mqttCustom`.
* `headers`, `qs` and `method`cannot be used, as they doesn’t have equivalence in MQTT
* Macro replacement is performed in `topic` and `payload` fields. `url`, `qos`, `user` and `passwd` are fixed values
```

Reply:
The approach before the doc rework is based in using HTTP description as a base and explain MQTT as an "amendment" to that base. However, it would be better a more homogeneous description taking both into account.

Difficult to explain what I have in mind :) I'll take care of this, in a specific commit in the PR